src/common/DisabledButtonInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode, useEffect } from 'react'
+import Tooltip, { TooltipProps } from '@mui/material/Tooltip'
+
+function DisabledButtonInfo({
+  conditions,
+  placement,
+  children,
+}: {
+  conditions: string[]
+  placement?: TooltipProps['placement']
+  children: ReactNode
+}) {
+  const [titleText, setTitleText] = React.useState<string>('')
+
+  useEffect(() => {
+    if (conditions !== undefined) {
+      let updatedTitleText = ''
+      conditions.forEach((condition) => {
+        if (condition !== '') {
+          updatedTitleText = `${updatedTitleText} ${condition}`
+        }
+      })
+      setTitleText(updatedTitleText)
+    }
+  }, [conditions])
+
+  return (
+    <Tooltip title={titleText} arrow placement={placement !== undefined ? placement : 'right'}>
+      <span>{children}</span>
```

Comment:
I don't think it actually makes a difference in this case, but I prefer to keep `span` for when we specifically want something to be displayed inline. As this is a generic button wrapper I'd consider using a `div` instead

Reply:
I agree, just used <div /> as that's what people suggested using for this workaround. Changed it!